src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -164,49 +173,55 @@ const ActiveState = ({
-                                    onShowReplies={onShowRepliesHandler(item.id, item.type)}
-                                    repliesTotalCount={item.total_reply_count}
-                                    replies={item.replies}
-                                    translations={translations}
-                                >
-                                    {hasNewThreadedReplies ? (
-                                        <BaseComment
-                                            {...item}
-                                            currentUser={currentUser}
-                                            getAvatarUrl={getAvatarUrl}
-                                            getMentionWithQuery={getMentionWithQuery}
-                                            getUserProfileUrl={getUserProfileUrl}
-                                            mentionSelectorContacts={mentionSelectorContacts}
-                                            onDelete={onCommentDelete}
-                                            onEdit={onCommentEdit}
-                                            onSelect={onCommentSelectHandler(item.id)}
-                                            permissions={{
-                                                can_delete: getProp(item.permissions, 'can_delete', false),
-                                                can_edit: getProp(item.permissions, 'can_edit', false),
-                                                can_reply: getProp(item.permissions, 'can_reply', false),
-                                                can_resolve: getProp(item.permissions, 'can_resolve', false),
-                                            }}
-                                            translations={translations}
+                                {hasNewThreadedReplies ? (
+                                    <BaseComment
+                                        {...item}
+                                        {...commentProps}
+                                        onDelete={onCommentDelete}
+                                        onEdit={onCommentEdit}
+                                        onSelect={onCommentSelectHandler(item.id)}
+                                        permissions={{
+                                            can_delete: getProp(item.permissions, 'can_delete', false),
+                                            can_edit: getProp(item.permissions, 'can_edit', false),
+                                            can_reply: getProp(item.permissions, 'can_reply', false),
+                                            can_resolve: getProp(item.permissions, 'can_resolve', false),
+                                        }}
+                                    >
+                                        <Replies
+                                            {...commentProps}
+                                            hasReplies={hasReplies}
```

Comment:
Do we need to add hasReplies here? If it renders this doesn't that mean that hasReplies is already true enabled also the hasNewThreadedReplies should make this unnecessary right?